tests/openassetio-core-c/private/hostApi/ManagerTest.cpp

```diff
@@ -159,14 +159,11 @@ SCENARIO("A host calls Manager::identifier") {
         openassetio::makeShared<MockManagerInterface>();
     auto& mockManagerInterface = static_cast<MockManagerInterface&>(*mockManagerInterfacePtr);
     // Create a HostSession with our mock HostInterface
-    managerApi::HostSessionPtr hostSessionPtr =
-        openassetio::makeShared<openassetio::managerApi::HostSession>(
-            openassetio::makeShared<managerApi::Host>(
-                openassetio::makeShared<MockHostInterface>()));
+    managerApi::HostSessionPtr hostSessionPtr = managerApi::HostSession::make(
+        managerApi::Host::make(openassetio::makeShared<MockHostInterface>()));
```

Comment:
[consider] We should probably remove the `makeShared` alias at this point.

Reply:
Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/495/commits/eda7e8bb2c04f8c5f1a365f95a32c7a1fd4e71a5).